components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -3610,9 +3610,19 @@ def activities(request, conn=None, **kwargs):
                 continue  # ignore
             if status not in ("failed", "finished"):
                 logger.info("Check callback on script: %s" % cbString)
-                proc = omero.grid.ScriptProcessPrx.checkedCast(
-                    conn.c.ic.stringToProxy(cbString))
-                cb = omero.scripts.ProcessCallbackI(conn.c, proc)
+                try:
+                    proc = omero.grid.ScriptProcessPrx.checkedCast(
+                        conn.c.ic.stringToProxy(cbString))
+                    cb = omero.scripts.ProcessCallbackI(conn.c, proc)
+                except Exception, x:
```

Comment:
Could you please use the same logic as in delete and wrap the entire code in try/catch block?

```
try:
    proc = omero.grid.ScriptProcessPrx.checkedCast(
    everything...
except Ice.ObjectNotExistException:
except Exception, x:
```